### ISSUE TITLE
breaks functionality of merge-conflicts package

### ISSUE THREAD

**Issue by SerbanC** [sindresorhus/atom-editorconfig#65]
Not sure if this is because of this package or the merge-conflicts package, but having atom-editorconfig enabled breaks functionality for [merge-conflicts](https://github.com/smashwilson/merge-conflicts/issues/196).

This is a file that merge-conflicts detects as having a conflict with editorconfig disabled:
<img width="1060" alt="screen shot 2016-02-16 at 15 57 10" src="https://cloud.githubusercontent.com/assets/1150246/13078739/723f49de-d4c9-11e5-8417-5c4d40f990e3.png">

This is the same file with editorconfig enabled:
<img width="337" alt="screen shot 2016-02-16 at 15 57 54" src="https://cloud.githubusercontent.com/assets/1150246/13078744/7457c2b4-d4c9-11e5-8353-b16b0a7542cc.png">

My full list of packages is:

```
emmet
file-icons
highlight-selected
imdone-atom
language-babel
language-nunjucks
language-vue
linter
linter-eslint
merge-conflicts
pigments
tabs-to-spaces
editorconfig
```

Editorconfig is the only package that breaks merge-conflicts, checked each one in the list against it.

**Comment by Arcanemagus** [comment]
So running the `Merge Conflicts: Detect` command from the command palette doesn't work for you? It works perfectly fine with both of those installed for me. (Or is it supposed to auto-detect and I've been missing out all this time? :stuck_out_tongue:)

**Comment by SerbanC** [comment]
@Arcanemagus the screenshots are after running `detect`, on a file that Merge Conflicts detected as having conflicts.

**Comment by Arcanemagus** [comment]
Before running `Merge Conflicts: Detect`:
![image](https://cloud.githubusercontent.com/assets/427137/13094350/f094e9a2-d4be-11e5-86b5-34758763b223.png)

After:
![image](https://cloud.githubusercontent.com/assets/427137/13094356/fe521448-d4be-11e5-9d1d-467cc4f29986.png)

![image](https://cloud.githubusercontent.com/assets/427137/13094374/1bb8b08c-d4bf-11e5-8806-4b9eb4544685.png)
Atom 1.6.0-beta3

If you backup your current profile, create a new one with just those two packages installed, do you still see this issue?

**Comment by smashwilson** [comment]
I'm wondering if it might have to do with the order in which our `atom.workspace.observeTextEditors()` callbacks are fired? None of the text modifications from editorconfig _look_ like they'd prevent the regexps from matching, but...

**Comment by florianb** [comment]
This bug also belongs to #69.

As @smashwilson pointed out i think this is caused by at least a wrong order event-processing. This seems to be a though nut to solve - i beg for patience. :sleeping:

**Comment by simon-ls** [comment]
@florianb I'm also getting this issue, as well #69

**Comment by florianb** [comment]
I believe this issue belongs to the already solved #69 whose fix will be released today. I therefore close this issue for now.

If the version `1.2.5` does not solve that bug for please come back and request a reopen!

Thank you very much! 👍
